testdata/p4_16_samples_outputs/issue561-midend.p4

```diff
@@ -21,12 +21,12 @@ control c(out bit<32> x) {
         x = u.h1.f + u.h2.g;
         u.h1.setValid();
         u.h1.f = 32w0;
-        x = x + u.h1.f;
+        x = x + 32w0;
```

Comment:
This is a sign we could do more strength reductions after this.

Reply:
Yes, which would then uncover more copy propagation opportunities.